src/com/powerlink/EplWrapper.h

```diff
@@ -18,90 +18,93 @@
 #include <fortelist.h>
 #include <forte_sync.h>
 
-struct SEplMapping{
-    struct SEplMappingValues{
-        unsigned int mDataSize;
-        unsigned int mPiOffset;
-        unsigned int mBitOffset;
-        char* mCurrentValue;
-
-        SEplMappingValues(unsigned int paDataSize, unsigned int paPiOffset, unsigned int paBitOffset) :
-            mDataSize(paDataSize), mPiOffset(paPiOffset), mBitOffset(paBitOffset){
-          mCurrentValue = new char[paDataSize];
-          for(unsigned int i = 0; i < paDataSize; i++){
-            mCurrentValue[i] = 0x00;
-          }
-        }
-
-        ~SEplMappingValues(){
-          delete[] mCurrentValue;
-        }
-
-      private:
-        SEplMappingValues(const SEplMappingValues &obj);
-        SEplMappingValues& operator=(const SEplMappingValues &obj);
-    };
-
-    typedef CSinglyLinkedList<SEplMappingValues*> TEplMappingList;
-    TEplMappingList mCurrentValues;
-
-    ~SEplMapping(){
-      while(!mCurrentValues.isEmpty()){
-        delete *(TEplMappingList::Iterator) mCurrentValues.begin();
-        mCurrentValues.popFront();
+struct EplMapping {
+  struct EplMappingValues {
+    unsigned int mDataSize;
+    unsigned int mPiOffset;
+    unsigned int mBitOffset;
+    char *mCurrentValue;
+
+    EplMappingValues(unsigned int pa_nDataSize, unsigned int pa_nPiOffset,
```

Comment:
we are not useing the type indicators anymore and where possible we are removing them. Therefore please don't add them anymore.

Reply:
I have removed the type indicators